keps/sig-apps/4368-support-managed-by-for-batch-jobs/README.md

```diff
@@ -1020,7 +1021,7 @@ A generic name without support across all k8s APIs might be confusing to the
 users, and supporting it for all k8s APIs would be much bigger effort than
 currently needed for the MultiKueue scenario use.
 
-The "managed-by" label idea has significant risks, such as
+The "managedBy" label idea has significant risks, such as
```

Comment:
```suggestion
The "managedBy" field idea has significant risks, such as
```

?

Reply:
No, this paragraph in the "Alternative names for label" refers to difference choices of the name labels. I revert "managedBy" to "managed-by" also here.